crates/interpreter/test.sh

```diff
@@ -28,3 +29,4 @@ cargo check --target=riscv32imc-unknown-none-elf \
 RUSTFLAGS=--cfg=portable_atomic_unsafe_assume_single_core \
   cargo check --target=riscv32imc-unknown-none-elf
 cargo test --features=debug,toctou,float-types,vector-types
+cargo test --features=debug,toctou,float-types,vector-types,threads
```

Comment:
Let's always test with threads. Otherwise we should also test without floats.
```suggestion
cargo test --features=debug,toctou,float-types,vector-types,threads
```